### ISSUE TITLE
Test failures with with Test-Simple-1.302014_001

### ISSUE THREAD

**Issue by eserte** [clustericious/Test-Clustericious-Blocking#1]
t/cluster.t fails if the latest Test-Simple is installed:

```
Can't locate object method "builder" via package "Test::More" (perhaps you forgot to load "Test::More"?) at /home/cpansand/.cpan/build/2016042221/Test-Simple-1.302014_001-x9OqBS/blib/lib/Test/More.pm line 166.
BEGIN failed--compilation aborted at t/cluster.t line 18.
Use of uninitialized value $$ in numeric ne (!=) at /home/cpansand/.cpan/build/2016042221/Test-Simple-1.302014_001-x9OqBS/blib/lib/Test2/API/Instance.pm line 413.
Use of uninitialized value $$ in exists at /opt/perl-5.18.4/lib/site_perl/5.18.4/x86_64-linux/forks.pm line 1464.
Use of uninitialized value $$ in numeric eq (==) at /opt/perl-5.18.4/lib/site_perl/5.18.4/x86_64-linux/forks.pm line 1464.
Use of uninitialized value $$ in exists at /opt/perl-5.18.4/lib/site_perl/5.18.4/x86_64-linux/forks.pm line 1505.
Use of uninitialized value $$ in numeric eq (==) at /home/cpansand/.cpan/build/2016042221/Test-Simple-1.302014_001-x9OqBS/blib/lib/Test2/IPC/Driver/Files.pm line 273 during global destruction.
t/cluster.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

@exodist: FYI

**Comment by exodist** [comment]
I am unable to reproduce this failure locally, possibly due to perl version or a version of something else in the chain.

**Comment by plicease** [comment]
I wasn't able to reproduce it either, can you include the output of `t/00_diag.t`?

**Comment by eserte** [comment]
This perl has older versions of Clustericious and Test::Clustericious::Cluster installed:

```
$ perl5.18.4 -e 'require Clustericious; require Test::Clustericious::Cluster; warn "$Clustericious::VERSION $Test::Clustericious::Cluster::VERSION"'
1.16 0.28 at -e line 1, <DATA> line 2125.
```

If these are upgraded, then the test does not fail anymore.

**Comment by exodist** [comment]
I will note that in the x-breaks metadata for Test-Simple.

**Comment by plicease** [comment]
I am upping the prereqs for the test in 0.03.  Those versions of Clustericious and TCC don't even work with the current version of Mojo anyway.

**Comment by plicease** [comment]
@exodist I don't think this diagnostic is right:

```
t/zzz-check-breaks.t ........................... # You have the following modules installed, which are not compatible with the latest Test::More:
# Installed version (0.31) of Test::Clustericious::Cluster is in range '<= 0.31'
```

TCC 0.31 seems to work fine, with 1.302014_009, it was the older version of TCC that demonstrated the problem.

**Comment by exodist** [comment]
@plicease ah, I made the mistake of thinking that since it wouldn't install last time I checked that it was still broken. Clearly I missed that it does not work on the latest mojo. I will put a fix for this in a branch that I will apply right after we go stable. I can't just mint a new dev release without resetting the release cycle, and I don't think this diagnostic is a big enough issue to reset the cycle over. But once the current dev is made stable I will start a new dev cycle with the fix.
